server/athenian/api/ariadne.py

```diff
@@ -1,14 +1,14 @@
-from pathlib import Path
 import traceback
+from pathlib import Path
 from typing import Any, Callable, Iterable, Optional
 
 import aiohttp.web
+import sentry_sdk
```

Comment:
Why did it put sentry_sdk here?

Reply:
by default pure import before from ..., configured as before now